### PR TITLE
#2705 log info from accelerator commands

### PR DESCRIPTION
**Pull request by mchoubby**

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java [hunk 1]
```diff
@@ -184,13 +187,24 @@ private void processComponentExchange(Component sourceContainer, Component sourc
     if ((componentExchange.getKind() == ComponentExchangeKind.DELEGATION)
         || (componentExchange.getKind() == ComponentExchangeKind.UNSET)
         || doesNodeAlreadyHaveAPhysicalLinkForComponentExchange(sourceContainer, componentExchange)) {
+      Status status = new Status(IStatus.ERROR, Activator.PLUGIN_ID, "Component exchange " + componentExchange.getName()
+          + " already have a physical link or the kind is not suitable.");
```
Comment (irina-toma) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java:191]: small typo: already has

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java [hunk 2]
```diff
@@ -360,6 +374,10 @@ protected void doCreatePhysicalLink(ComponentExchange componentExchange, Compone
 
     exchangeInput.getOwnedPhysicalLinks().add(physicalLink);
     exchangeOutput.getOwnedPhysicalLinks().add(physicalLink);
+    Status status = new Status(IStatus.INFO, Activator.PLUGIN_ID,
+        "The Physical link " + physicalLink.getName() + " has been succefully created between the source "
+            + exchangeInput.getLabel() + " and target " + exchangeOutput.getLabel());
```
Comment (irina-toma) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/PhysicalLinksCreator.java:379]: "and the target"

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/ComponentExchangesCreator.java [hunk 3]
```diff
@@ -34,120 +37,138 @@
...
+    // source side delegation
+    InformationsExchanger target = componentExchange_p.getTarget();
+    createComponentPortAllocation(target, inP);
+
+    // target side Delegation
+    InformationsExchanger source = componentExchange_p.getSource();
+    createComponentPortAllocation(source, outP);
+    Status status = new Status(IStatus.INFO, Activator.PLUGIN_ID,
+        "The Physical link " + physicalLink.getName() + " has been succefully created between the source "
+            + exchangeInput_p.getLabel() + " and target " + exchangeOutput_p.getLabel());
```
Comment (irina-toma) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/ComponentExchangesCreator.java:125]: "and the target"

**Review by irina-toma — CHANGES_REQUESTED** [review]

**Review by irina-toma — APPROVED** [review]

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/DefaultExchangesCreator.java [hunk 4]
```diff
@@ -171,17 +196,25 @@ protected void doCreateExchange(FunctionalExchange functionalExchange_p, Compone
...
     // Creates the exchange allocation
-    ComponentExchangeFunctionalExchangeAllocation cfea = FaFactory.eINSTANCE.createComponentExchangeFunctionalExchangeAllocation();
+    ComponentExchangeFunctionalExchangeAllocation cfea = FaFactory.eINSTANCE
+        .createComponentExchangeFunctionalExchangeAllocation();
     cfea.setSourceElement(ce);
     cfea.setTargetElement(functionalExchange_p);
     ce.getOwnedComponentExchangeFunctionalExchangeAllocations().add(cfea);
+
+    Status status = new Status(IStatus.INFO, Activator.PLUGIN_ID,
+        "The Component exchange " + ce.getName() + " has been succefully created between the exchange input component "
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/DefaultExchangesCreator.java:215]: Can you make information message navigable towards the elements 
(using EmbeddedMessage in the status like below: https://github.com/eclipse/capella/blob/5dfdd588f90ee4bac0cd803e47d3bf065be809e4/core/plugins/org.polarsys.capella.core.projection.interfaces/src/org/polarsys/capella/core/projection/interfaces/generateInterfaces/Rule_FunctionPort.java#L93

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/EntityExchangesCreator.java [hunk 5]
```diff
@@ -203,10 +223,15 @@ protected void createCommunicationMean(FunctionalExchange functionalExchange_p,
...
     CapellaElementExt.creationService(ce);
 
-    ComponentExchangeFunctionalExchangeAllocation allocation = FaFactory.eINSTANCE.createComponentExchangeFunctionalExchangeAllocation();
+    ComponentExchangeFunctionalExchangeAllocation allocation = FaFactory.eINSTANCE
+        .createComponentExchangeFunctionalExchangeAllocation();
     ce.getOwnedComponentExchangeFunctionalExchangeAllocations().add(allocation);
     allocation.setTargetElement(functionalExchange_p);
     allocation.setSourceElement(ce);
     CapellaElementExt.creationService(allocation);
+    Status status = new Status(IStatus.INFO, Activator.PLUGIN_ID,
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/EntityExchangesCreator.java:232]: same here

**Review by pdulth — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/ComponentExchangesCreator.java [hunk 6]
```diff
@@ -34,120 +36,138 @@
...
+    createComponentPortAllocation(target, inP);
+
+    // target side Delegation
+    InformationsExchanger source = componentExchange_p.getSource();
+    createComponentPortAllocation(source, outP);
+    String message = "The Physical link " + physicalLink.getName() + " has been succefully created between the source "
+        + exchangeInput_p.getLabel() + " and the target " + exchangeOutput_p.getLabel();
+    EmbeddedMessage eMessage = new EmbeddedMessage(message, logger.getName(),
+        Arrays.asList(physicalLink, exchangeInput_p, exchangeOutput_p));
+    logger.error(eMessage);
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.projection.exchanges/src/org/polarsys/capella/core/projection/exchanges/ComponentExchangesCreator.java:126]: Is it an error? or an info

**Review by pdulth — CHANGES_REQUESTED** [review]

**Review by pdulth — APPROVED** [review]

**Pull request merged**